sql/testSelectQueries.sql

```diff
@@ -0,0 +1,13 @@
+
+-- List tasks and users by user;
+SELECT CONCAT(dbo.users.lastName, ',', dbo.users.firstName) AS userName,
+	dbo.tasks.title AS Title,
+	dbo.tasks.taskDescription AS "Description"
```

Comment:
```suggestion
	dbo.tasks.taskDescription AS Description
```

Reply:
I wrote the alias "Description" with the quotation marks because apparently Description is apparently a reserved word or keyword in Transact-SQL (my IDE highlighted it anyway).  Writing in quotes is a way of getting around having reserved words as an alias (in SQL in general.)

Reply:
oooooo

Reply:
dismiss this then